demos/demo_solo12_from_yaml.cpp

```diff
@@ -45,30 +46,35 @@ int main()
             {
                 if (!is_calibrated)
                 {
-                    is_calibrated = calib_ctrl->Run();
+                    is_calibrated = calib_ctrl->RunAndGoTo(pos_init);
                     if (is_calibrated)
                     {
                         std::cout << "Calibration is done." << std::endl;
                     }
                 }
                 else
                 {
-                    // Run the main controller.
-                    auto pos = robot->joints->GetPositions();
-                    auto vel = robot->joints->GetVelocities();
-                    // Reverse the positions;
-                    for (int i = 0; i < 12; i++)
-                    {
-                        torques[i] = -kp * pos[i] - kd * vel[i];
-                    }
-                    robot->joints->SetTorques(torques);
+                    // Set all command quantities
+                    robot->joints->SetTorques(Eigen::VectorXd::Zero(12));
+                    robot->joints->SetDesiredPositions(pos_init);
+                    robot->joints->SetDesiredVelocities(
+                        Eigen::VectorXd::Zero(12));
```

Comment:
This allocates a 12-dim zero vector in each loop. Please create a local variable like `zero_velocity_12` and initialize it before the main control loop starts.

Reply:
I will do it :+1: